### PR TITLE
Milestone:Portfolio: desktop version

### PR DESCRIPTION
**Pull request by ibrahim777764**
### Milestone:Portfolio: desktop version 

- understood how to create Is adaptable to different screen sizes using media queries. 
- understood how to parse a Figma design to create a UI

### CODE REVIEW

**Comment by ibrahim777764** [comment]
please review my code, let me know if there is any changes.thankyou

File: index.html [hunk 1]
```diff
@@ -1,278 +1,347 @@
...
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>ibrahim's portfolio</title>
+    <title>ibrahim's Portfolio</title>
     <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
     <header>
```
Comment (Deepakdanger) [inline index.html:17]: - [ ] While doing the **desktop version**, you mismatch the **mobile version** of the header section. Kindly fix that.
- Your website
fdbfbdf
- Figma Design 
nfgnfgn

File: index.html [hunk 2]
```diff
@@ -1,278 +1,347 @@
...
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <button type="button" class="btn-popup">
+                    See Project
+                </button>
+            </article>
+        </div>
+    </section>
+    <section class="container-flex about-myself" id="about-myself-id">
```
Comment (Deepakdanger) [inline index.html:195]: - [ ] While doing the **desktop version**, you mismatch the **mobile version** of the `About me` section. Kindly fix that.
- Your website

sdvdv

- Figma website

dbfdfbd

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @ibrahim777764, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ibrahim777764** [comment]
### Updated all required changes

please review my code thankyou

**Review by medaminedev66 — APPROVED** [review]
Hi @ibrahim777764 , 

# Status: Approved ✔️
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**